### PR TITLE
refactor: removed unnused code

### PR DESCRIPTION
**Pull request by albertolive**
In the `CustodyController` institutional package, we have two methods, `getCustodianConnectRequest` and `setCustodianConnectRequest`, that are not being used in the monorepo, so this ticket aims to remove them.

Ticket: https://consensyssoftware.atlassian.net/browse/MMI-4578

### CODE REVIEW

File: packages/custodyController/src/custody.ts [hunk 1]
```diff
@@ -104,34 +104,6 @@ export class CustodyController {
     return custodyTypes;
   }
 
-  // TODO - Can probably be removed as that state is not being used anywhere
-  setCustodianConnectRequest({
-    token,
```
Comment (zone-live) [inline packages/custodyController/src/custody.ts:109]: I think setCustodianConnectRequest is being used in the extension. https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.js#L79
Also getCustodianConnectRequest is being referenced there, but doesn't seem to be used in the `interactive-replacement-token-page.js`

**Review by zone-live — CHANGES_REQUESTED** [review]
Just double check if in the extension is not being used 🙏🏼

File: packages/custodyController/src/custody.ts (hunk 1 above)
Comment (albertolive) [reply packages/custodyController/src/custody.ts:109]: I removed them from the extension as we don't need them:
https://github.com/MetaMask/metamask-extension/pull/23000

**Comment by albertolive** [comment]
> Also, if you look closely the file `ExtensionActions` with the `mmiActionsFactory` can be removed, since these actions are in the extension under `/institution-background.ts`. 👍🏼

Are you saying that I can remove all the file?

**Review by zone-live — APPROVED** [review]

**Pull request merged**